docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM debian:jessie
+
+ARG DEBIAN_FRONTEND="noninteractive"
+
+ENV LANG C.UTF-8
```

Comment:
Not sure why that's needed either, so adding a short comment would be helpful if something doesn't work without it.